docs/data-sources/k8_clusters.md

```diff
@@ -0,0 +1,71 @@
+---
+subcategory: "Compute Engine"
+layout: "ionoscloud"
+page_title: "IonosCloud : k8s_clusters"
```

Comment:
```suggestion
page_title: "IonosCloud : ionoscloud_k8s_clusters"
```

Reply:
modified ([b9a7d92](https://github.com/ionos-cloud/terraform-provider-ionoscloud/pull/519/commits/b9a7d92cfbcc64f79aa318a0eeab6fc491584516))

Reply:
you can just commit suggestion next time, should be easier.